example/scalalib/publishing/9-cosmo-assembly/build.mill

```diff
@@ -0,0 +1,20 @@
+//// SNIPPET:BUILD
```

Comment:
No need these `SNIPPET` directives since we're not sharing any code with other templates